src/main/scala/org/constellation/API.scala

```diff
@@ -69,7 +69,10 @@ object ProcessingConfig {
     maxMemPoolSize = 1000,
     minPeerTimeAddedSeconds = 5,
     roundsPerMessage = 1,
-    leavingStandbyTimeout = 3
+    leavingStandbyTimeout = 3,
+    recentSnapshotNumber =
+    ConfigUtil.constellation.getInt("snapshot.snapshotHeightRedownloadDelayInterval")
```

Comment:
I said on our meetings, that if we gonna do the redownload using the new flow you've introduced and redownload interval is hovering around the majority, this change will not work and we should keep track of more snapshots than redownload delay interval. Otherwise, node who will be below/above interval, will be no able to make a redownload.